internal/component/prometheus/exporter/cloudwatch/config.go

```diff
@@ -26,6 +28,7 @@ var defaults = Arguments{
 		Enabled:        false,
 		ScrapeInterval: 5 * time.Minute,
 	},
+	AWSSDKVersion: "v2",
```

Comment:
I think we should default to v1 for now to prevent breaking anyone on upgrade. We can make v2 the default following YACE changing.

Reply:
Let's do it 👍